### DISCUSSION TITLE
Prisma won't work after build sveltekit tauri app

### DISCUSSION THREAD

**Discussion by islamzaoui** [prisma/prisma#20103] [Q&A]
hello guys
I created a **Prisma SQLite** database for my **svelte-kit Tauri app**, and it works fine when developing. Still, after building my app when I run the executable and go to the page that uses Prisma to fetch data from dev.db the page froze and nothing happens, I can navigate to other pages that don't use Prisma, how I can fix this problem?
this my the code that I use to request data from SQLite db all over **+page.server.ts / +server.ts**:

```
import PrismaClinetPkg from "@prisma/client"

const PrismaClient = PrismaClinetPkg.PrismaClient
const prisma = new PrismaClient()

export {prisma}
```

_please give me a solution that won't make me rewrite the whole app._

**Comment by janpio** [comment]
What version of Prisma are you using?

Can you get any error message?
If not, can you create a minimal reproduction and put that on GitHub so we can poke at it ourselves?

**Reply by nurul3101** [reply]
@zicoding2 Did you had a chance to look at Jan's response? Please let us know in case you still need assistance.
If not, please consider marking the discussion as answered/closed.

**Reply by nurul3101** [reply]
We are closing this Discussion due to inactivity. Please open a new Discussion if you have a similar problem 🙏

**Comment by MachariaK** [comment]
Related: https://github.com/nuxt/nuxt/issues/22738

**Comment by MachariaK** [comment]
I was experiencing this exact problem. 

The Prisma Rust Client creator pointed me to the documentation: [https://prisma.brendonovich.dev/extra/migrations](url). 

I enabled the `migrations` feature for `prisma-client-rust` and `prisma-client-rust-cli` then ran` cargo run --bin prisma migrate deploy --schema=./prisma/schema.prisma`. (This will depend on your setup). 

Then added the first five lines in the main function: 

```
#[tokio::main]
async fn main() {
    let client = prisma::new_client().await.unwrap();
    #[cfg(debug_assertions)]
    client._db_push().await;
    #[cfg(not(debug_assertions))]
    client._migrate_deploy().await;

    tauri::Builder::default()
        .invoke_handler(tauri::generate_handler![greet])
        .run(tauri::generate_context!())
        .expect("error while running tauri application");
}
```
This resolved my problem and the app can now access the database in production.
